### PR TITLE
Remove https handling

### PR DESCRIPTION
**Pull request by coverbeck**
SEAB-465

Load balancer will handle SSL termination, so remove
https handling.

Needed to tweak robots.txt handling; the previous
way wasn't working for me.

Remove certs stuff from README

### CODE REVIEW

File: templates/default.nginx_http.conf.template [hunk 1]
```diff
@@ -1,14 +1,94 @@
...
+    ~image/                    604800;
+}
+
 server {
-    listen 80;
-    listen [::]:80;
-    server_name  {{ DOMAIN_NAME }};
+    server_name  default_server;
+    include /etc/nginx/conf.d/default.nginx_http.shared.conf;
+    # listen       4200 http2;
```
Comment (david4096) [inline templates/default.nginx_http.conf.template]: This seems like a possible issue. HTTP/2 wont work without ssl afaik. We might still SSL the connection to the load balancer if we want to support it. In general I favor supporting HTTP/2 if we can however having two separated SSL boundaries seems odd.

Comment (coverbeck) [reply templates/default.nginx_http.conf.template]: Currently the connection from the load balancer to here is over http, not https.

**Comment by coverbeck** [comment]
> Do you intend to move the cloudformation template into this repository once its fleshed out?

Not sure -- currently putting the cloud formation templates in a separate [repo](https://github.com/dockstore/dockstore-deploy). My initial thought is that we should keep AWS-isms out of this repo; if we or somebody else wants to run this on a different cloud, this would run anywhere, and then a cloud-specific proxy/load-balancer could be setup in front of it separately.

**Comment by coverbeck** [comment]
OTOH, we already have AWS logging in docker-compose.yml. Still, I'd be inclined to keep them separate for now.

**Pull request merged**